db/migrate/20211119154133_change_column_types.rb

```diff
@@ -0,0 +1,21 @@
+class ChangeColumnTypes < ActiveRecord::Migration[6.1]
+  def up
+    change_table :case_logs, bulk: true do |t|
+      t.change :first_time_property_let_as_social_housing, :string
```

Comment:
Given that this hasn't gone into main yet you can probably combine these migrations (i.e. just a single migration that creates these columns with the right types in the first place)?

Reply:
Done this all in one